rust/src/coordinator.rs

```diff
@@ -52,68 +48,16 @@ pub enum Phase {
     Sum2,
 }
 
-/// Events the protocol emits.
-#[derive(Debug, PartialEq)]
-pub enum ProtocolEvent {
-    /// The round starts with the given parameters. The coordinator is
-    /// now in the sum phase.
-    StartSum(RoundParameters),
-
-    /// The sum phase finished and produced the given sum
-    /// dictionary. The coordinator is now in the update phase.
-    StartUpdate(SumDict),
-
-    /// The update phase finished and produced the given seed
-    /// dictionary. The coordinator is now in the sum2 phase.
-    StartSum2(SeedDict),
-
-    /// The sum2 phase finished and produced a global model. The
-    /// coordinator is now back to the idle phase.
-    EndRound(Option<()>),
-}
-
-#[derive(AsRef, AsMut, Clone, Debug, PartialEq, Eq)]
-/// A seed for a round.
-pub struct RoundSeed(box_::Seed);
```

Comment:
what's the reason to revert this back to `Vec<u8>`? the initial idea was to wrap the sodiumoxide `Seed` to benefit from memory zeroing on drop, because this is a secret value.

Reply:
I didn't change it on purpose. I'll revert it back to `box_::Seed`